ui/src/table.test.tsx

```diff
@@ -1676,6 +1676,58 @@ describe('Table.tsx', () => {
       expect(container.querySelectorAll('.ms-GroupHeader-title')[0]).toHaveTextContent('1/20/1970, 4:58:47 AM(0)')
       expect(container.querySelectorAll('.ms-GroupHeader-title')[1]).toHaveTextContent('6/22/2022, 8:47:51 PM(1)')
     })
+
+    it('Collapses all group by list - fire event', () => {
```

Comment:
Why do we need "duplicate" tests? Wouldn't the ones in the "Groups" suite be enough?

Reply:
I thought these would be useful since the way to end up with these groups are different - explicitly defined vs group by. Even though this new 'group_change' event code is the same regardless, since the steps getting there are different it seemed worth having the tests to protect from future changes that might result in this breaking only for 'group by' - not sure what that would be now, but seems useful to protect against that. 

If you disagree, I can definitely remove the 'group by' tests, so just let me know!